vardbg/output/video_writer/getstyle.py

```diff
@@ -0,0 +1,55 @@
+"""
+    vardbg.styles
+    ~~~~~~~~~~~~~~~
+
+"""
+
+from pygments.util import ClassNotFound
+
+
+STYLE_ENTRY_POINT = 'vardbg.assets.styles'
+
+
+def iter_entry_points(group_name):
+    try:
+        import pkg_resources
+    except (ImportError, IOError):
+        return []
+
+    return pkg_resources.iter_entry_points(group_name)
+
+def find_plugin_styles():
+    for entrypoint in iter_entry_points(STYLE_ENTRY_POINT):
```

Comment:
No need for runtime scheme discovery here. For something simple like this, it's fine to just import everything in `__init__.py` in styles.